mincall/models/residual_deep.py

```diff
@@ -11,8 +11,7 @@
 from mincall.model_utils import Model
 load_dotenv(find_dotenv())
 
-
-def model_fn(net, X_len, max_reach, block_size, out_classes, batch_size, dtype, **kwargs):
+def model_fn(net: tf.Graph, X_len, max_reach, block_size, out_classes, batch_size, dtype, **kwargs):
```

Comment:
Promjenio si shink_factor u konfiguraciji, no u ovoj funkciji nisi promjenio broj max_poolova koji uzrokuju taj shrink_factor. To ti fali.